### ISSUE TITLE
Add jax support!

### ISSUE THREAD

**Issue by brianwa84** [brendanhasz/probflow#14]
TFP has a JAX-backed impl that's fairly functional at this point (note, some distributions are still coming together).
If you wanted to add support for running on JAX as well, it might just be a matter of `tfp = tfp.experimental.substrates.jax`.

**Comment by brendanhasz** [comment]
Good idea!  I'm thinking I might add JAX as a third backend.  Might not get to it for a while honestly but I've heard lots of good things about JAX so hopefully eventually!
